src/loaders/zarrLoader.js

```diff
@@ -92,24 +92,15 @@ export default class ZarrLoader {
         // big endian
         byteSwapInplace(data);
       }
-      const width = source.chunks[xIndex];
-      const height = source.chunks[yIndex];
-      if (height < this.tileSize || width < this.tileSize) {
-        return padTileWithZeros(
-          { data, width, height },
-          this.tileSize,
-          this.tileSize
-        );
-      }
       return data;
     });
-    const data = await Promise.all(dataRequests);
+    const tileData = truncateTiles(await Promise.all(dataRequests), this, {
+      x,
+      y,
+      z
+    });
```

Comment:
I don't think this is needed on the zarr loader (and am not sure why we also were padding before).

Reply:
The padding was to prevent the binding issues we were having with textures, I think.